cg/store/filters/status_order_filters.py

```diff
@@ -28,6 +28,19 @@ def filter_orders_by_ticket_id(orders: Query, ticket_id: int | None, **kwargs) -
     return orders.filter(Order.ticket_id == ticket_id) if ticket_id else orders
 
 
+def filter_orders_by_search(orders: Query, search: str | None, **kwargs) -> Query:
+    if search:
+        orders = orders.join(Order.customer)
+        orders.filter(
+            or_(
+                Order.id.startswith(search),
+                Customer.name.ilike(f"%{search}%"),
```

Comment:
Probably more relevant with internal id? Maybe both would be nice? cust000 etc are the internal_ids though

Reply:
This is true